src/widgets/FormControl.js

```diff
@@ -9,7 +9,7 @@ import { connect } from 'react-redux';
 import { PageButton } from './PageButton';
 
 import { FORM_INPUT_TYPES } from '../utilities/formInputConfigs';
-import { FormTextInput } from './FormInputs/FromTextInput';
+import { FormTextInput } from './FormInputs/FormTextInput';
```

Comment:
Renamed this file :(

Reply:
Sneaky when the misspelling is a valid spelling heh